src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -452,9 +458,6 @@ bool InputStateMachineEngine::ActionCsiDispatch(const VTID id, const VTParameter
     case CsiActionCodes::CursorBackTab:
         _WriteSingleKey(VK_TAB, SHIFT_PRESSED);
         return true;
-    case CsiActionCodes::DTTERM_WindowManipulation:
-        _pDispatch->WindowManipulation(parameters.at(0), parameters.at(1), parameters.at(2));

```

Comment:
Does this mean we can unimplement `InteractDispatch::WindowManipulation` as well?

Reply:
It's still used in `OutputStateMachineEngine` isn't it?

Reply:
Right, but that goes through `adaptDispatch`.

The one from ISM goes through `InteractDispatch`.

And upon looking at `InteractDispatch::WindowManipulation`, I fear that we actually might be using it (window manipulation) to help conpty's window transition through states ......